hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/SynthTxnUtils.java

```diff
@@ -119,7 +119,14 @@ public static ContractCreateTransactionBody synthContractCreationFromParent(
         return builder.build();
     }
 
-    private static boolean hasNonDegenerateAutoRenewAccountId(@NonNull final Account account) {
+    /**
+     * Returns whether the given account has an auto-renew account id that is not
+     * {@code 0.0.0}.
+     *
+     * @param account the account
+     * @return whether the given account has an auto-renew account id that is not
+     */
+    public static boolean hasNonDegenerateAutoRenewAccountId(@NonNull final Account account) {
```

Comment:
In a different world this could go somewhere else where it might be more generally useful, e.g., in Account?

Reply:
Hmm true.